src/loader/esm.ts

```diff
@@ -0,0 +1,3 @@
+import { createRequire } from "module";
+export * from "@esbuild-kit/esm-loader";
+createRequire(import.meta.url)("@esbuild-kit/cjs-loader");
```

Comment:
FYI with pkgroll, you should be able to just do `require('@esbuild-kit/cjs-loader')`

Reply:
The idea is that this is loaded from nodes esm environment, but still adds the cjs require hook so that if you manually createRequire it will still work. I don't think the above suggestion would work?

Reply:
I guess it should be fine to import a cjs module which should also work. Never mind!

Reply:
It will polyfill it:

https://github.com/privatenumber/pkgroll#require-in-esm

Reply:
Should we even need that though? In theory this would just an import to a cjs module which _should_ just work right?

In theory I can just have

```js
export * from "@esbuild-kit/esm-loader";
import "@esbuild-kit/cjs-loader";
```